initialize/initialize.go

```diff
@@ -145,7 +156,7 @@ func githubSearch(query string) ([]Repo, error) {
 
 		incompleteResults := result.GetIncompleteResults()
 		if incompleteResults {
-			log.Println("WARNING: Github API timed out before completing query")
+			log.Println("WARNING: Github API timed out before completing query. Querying again for additional results.")
```

Comment:
I'm confused by this behavior and log message. I would think `incompleteResults` just means that we need to keep paginating... can we just remove this WARNING statement altogether? The next log line explains that we're paginating

https://godoc.org/github.com/google/go-github/github#IssuesSearchResult

Reply:
I was trying to make it less confusing with the extra words, but your solution of just removing that line is better.